Volha_Ivanova/Module_1_Java/Classes_and_Objects/src/com/epam/cdp/volha_ivanova/task1/sweets/Gift.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.cdp.volha_ivanova.task1.sweets;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+
+import com.epam.cdp.volha_ivanova.task1.sweets.candies.chocolateCandies.ChocolateCandies;
+import com.epam.cdp.volha_ivanova.task1.sweets.candies.lollipop.Lollipop;
+import com.epam.cdp.volha_ivanova.task1.sweets.chocolate.Chocolate;
+import com.epam.cdp.volha_ivanova.task1.sweets.waffles.Waffles;
+
+public class Gift {	
+	
+	int giftWeight;			
```

Comment:
First of all. access modifiers. I've written about that in previous task. each field in class should have correct access modifier. usually it's private (according to OOP principle).